Package.swift

```diff
@@ -1,15 +1,22 @@
-// swift-tools-version:4.2
+// swift-tools-version:5.7
 import PackageDescription
 
 let package = Package(
     name: "TPKeyboardAvoiding",
+    platforms: [
+        .iOS(.v12)
+    ],
```

Comment:
Why iOS 12 was chosen? Maybe we could go with iOS 14? Since marketplace support only 14.3 and up?

Reply:
I took the lowest available option, since the code is not requiring any newer APIs available in later version of iOS.
So its unrelated to the minimum version of what Vinted supports, it should represent what the package needs are inregards to available apis. 
But if you think its better to have 14, i can do that as well...

Reply:
It's fine, not worth the change, just curious for selected version

Your argument is good, although I doubt there will be any other users of this framework besides us :D

Reply:
True, but its not about the minimum version the consumers of the dependency has, its about the minimum iOS version the dependency needs for it to compile. 

As it seems, the usage of TPKeyboardAvoiding is now down to a single screen, the delete item reason screen will be removed soon 🎉 . I will focus on removing the dependency from the contact-us view-controller and we no longer will need it. So probably will close this PR 😃